flatgeobuf/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_library(flatgeobuf STATIC flatgeobuf_c.cpp geometryreader.cpp packedrtree.cpp)
```

Comment:
I would remove that file, and set a flatgeobuf_SOURCES variables in top CMakeLists.txt, and do like agg_SOURCES to incorporate it to the mapserver library

Reply:
thanks @rouault, it makes sense I think :)